### PR TITLE
Extract error and loading state to single implementation

### LINKED ISSUE #145: Extract reusable error state component
See #136 and #144 . These have an error state component that replaces view contents when some kind of application error occurs, ex. the connection to the target JVM fails. This error state component should be refactored and extracted out so that all uses share a common implementation, rather than a copy-paste of the same template.

### PR DESCRIPTION
**Pull request by Alexjsenn**
Fixes #145 and #146

### CODE REVIEW

**Comment by Alexjsenn** [comment]
The naming is not ideal, maybe it should be LoadingView for consistency?

File: src/app/ErrorView/ErrorView.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import { Bullseye, Text } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+
+export interface ErrorViewProps {
+    message: string;
+}
+
+export const ErrorView: React.FunctionComponent<ErrorViewProps> = (props) => {
+    return (<>
+        <br/>
```
Comment (andrewazores) [inline src/app/ErrorView/ErrorView.tsx:48]: Indentation here looks funny

File: src/app/Loading/Loading.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import * as React from 'react';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+
+export const Loading: React.FunctionComponent = () => {
```
Comment (andrewazores) [inline src/app/Loading/Loading.tsx:41]: I agree, `Loading` isn't the best name. `LoadingView` or `LoadingState` or `LoadingWidget`, anything along those lines might be good.

**Comment by andrewazores** [comment]
Looks great overall, nice cleanup.

**Review by andrewazores — APPROVED** [review]

**Pull request merged**